src/Orleans/CodeGeneration/TypeUtils.cs

```diff
@@ -397,7 +397,7 @@ public static bool IsGrainClass(Type type)
         public static bool IsSystemTargetClass(Type type)
         {
             Type systemTargetType;
-            if (!TryResolveType("Orleans.Runtime.SystemTarget, OrleansRuntime", out systemTargetType)) return false;
+            if (!TryResolveType("Orleans.Runtime.SystemTarget", out systemTargetType)) return false;
```

Comment:
If I remember correctly, because `SystemTarget` is defined in OrleansRuntime and not Orleans